src/content/docs/ts-sdk/props/transition.mdx

```diff
@@ -29,3 +30,11 @@ Easing functions are used to interpolate between two values over time during tra
 
 - **Type**: [`EasingFunction`](/ts-sdk/props/easing-function)
 - **Default value**: `"linear"`
+
+---
+
+### shouldInterrupt
+If `true`, on scene update a transition that is already in progress will be interrupted and a new transition will start from the current state.
```

Comment:
from sdk perspective, scene update is not a clear term, it is not clear what it does